device/cuda/include/cuda/seeding/track_params_estimation.hpp

```diff
@@ -0,0 +1,44 @@
+/** TRACCC library, part of the ACTS project (R&D line)
+ *
+ * (c) 2021 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
+
+#pragma once
+
+#include <cuda/seeding/track_params_estimating.cuh>
+
+namespace traccc {
+namespace cuda {
+
+/// track parameter estimation for cuda
+class track_params_estimation {
```

Comment:
Right. I removed `track_params_estimating` since it is confusing as you mentioned. Instead, I directly implement operator() in `track_params_estimation.cu`. Now it will look much better.

For the algorithm part, I will explain in the below as well; I removed the `track_params_estimation_algorithm` as it seems not needed for the moment. The initial motivation of algorithm one is for setting the configuration parameter rather than touching the original track_parameter_estimation in core directory. but It is apparent that we don't need the algorithm version because there is not any configuration parameter for track parameter estimation for the moment.